masterlist.yaml

```diff
@@ -5460,3 +5484,33 @@ plugins:
       - Delev
       - Invent
       - Relev
+  - name: 'BandolierForNPC - Cloaks of Skyrim Patch.esp'
+    req:
+      - 'Dr_Bandolier.esp'
+      - 'Cloaks.esp'
```

Comment:
File masters automatically have a required entry, there is no need to add these as required if they're in the `esp` header.